models/form.js

```diff
@@ -4,14 +4,14 @@ var Schema = mongoose.Schema;
 var formSchema = Schema({
   _studentID: { type: Schema.Types.ObjectId, ref: 'Student' },
   period: String,
-  date: Date,
+  date: Number,
```

Comment:
I tried that but it kept erroring out..  It also doesn't really matter because the number method (which is time since a specific date in the past) is something that can easily be transfered in and out of Date objects in javascript.  I can try to change it back if you want but I don't see any huge functionality problems at this point